src/integration/java/io/pinecone/integration/dataPlane/UpsertAndQueryServerlessTest.java

```diff
@@ -211,26 +208,21 @@ public void upsertOptionalVectorsAndQueryIndexFutureTest() throws InterruptedExc
             Collections.sort(expectedSparseValues);
             Collections.sort(sparseValues);
             assertEquals(expectedSparseValues, sparseValues);
-        });
+        }, 4);
     }
 
     @Test
     public void upsertNullSparseIndicesNotNullSparseValuesFutureTest() {
         String id = RandomStringBuilder.build(3);
-        StringBuilder exceptionMessage = new StringBuilder();
         try {
             asyncIndex.upsert(id,
                     generateVectorValuesByDimension(dimension),
                     null,
                     generateVectorValuesByDimension(dimension),
                     null,
                     null).get();
-        } catch (PineconeValidationException validationException) {
```

Comment:
Same as the previous change similar to this, I think we can just use one `catch`.

Reply:
Same comment as prev